kernel/kernel-config-server/README.md

```diff
@@ -83,7 +83,11 @@ And place the encrypted value in client application properties file with the for
 
 `curl http://<your-config-server-address>/<application-context-path-if-any>/decrypt -d <encrypted-value-to-decrypt>`
 
-**NOTE** There is no need to write decryption mechanism in client applications for encrypted values. They will be automatically decrypted by config server. 
+**NOTE** 
+1. There is no need to write decryption mechanism in client applications for encrypted values. They will be automatically decrypted by config server. 
+2. Config server doesn't support asterisk (*) in URL as supported in previous java versions. We need to provide the complete URL while fetching any config file from the config server. 
```

Comment:
can you update the note: The Config Server no longer supports the use of an asterisk (*) in the URL as it did in previous Java versions. Now, when fetching a configuration file from the Config Server, you must provide the complete URL. For example, the URL format:

http://<config-server-url>/*/<profile>/<label>/<path-to-file>

should be changed to:

http://<config-server-url>/application/<profile>/<label>/<path-to-file>.

Reply:
as its not confirmed from release document so approved the pr